### PR TITLE
include padding when autoresizing multiline inputs

### PR DESCRIPTION
**Pull request by hyperobject**
Fixes an issue where the attachment caption text box would shrink to one line after typing in it due to its `minHeight` being less than `rowsMin` + padding. 

@keybase/react-hackers 
@keybase/hotpotatosquad

### CODE REVIEW

File: shared/common-adapters/plain-input.d.ts [hunk 1]
```diff
@@ -72,6 +72,7 @@ export type Props = {
   onBlur?: () => void
   onChangeText?: (text: string) => void
   onFocus?: () => void
+  padding?: keyof typeof globalMargins
```
Comment (chrisnojima) [inline shared/common-adapters/plain-input.d.ts]: Mobile will also have to support this

File: shared/chat/conversation/attachment-get-titles/index.tsx [hunk 2]
```diff
@@ -204,10 +205,10 @@ const styles = Styles.styleSheetCreate({
       borderWidth: 1,
       marginBottom: Styles.globalMargins.tiny,
       // RN wasn't obeying `padding: Styles.globalMargins.tiny`.
-      paddingBottom: Styles.globalMargins.tiny,
-      paddingLeft: Styles.globalMargins.tiny,
-      paddingRight: Styles.globalMargins.tiny,
-      paddingTop: Styles.globalMargins.tiny,
+      // paddingBottom: Styles.globalMargins.tiny,
```
Comment (chrisnojima) [inline shared/chat/conversation/attachment-get-titles/index.tsx]: :fire:

File: shared/chat/conversation/attachment-get-titles/index.tsx [hunk 3]
```diff
@@ -115,6 +115,7 @@ class GetTitles extends React.Component<Props, State> {
                 placeholder={titleHint}
                 multiline={true}
                 rowsMin={2}
+                padding="tiny"
```
Comment (chrisnojima) [inline shared/chat/conversation/attachment-get-titles/index.tsx]: when we hoist up a style into a prop we need to blacklist it. see https://github.com/keybase/client/blob/master/shared/common-adapters/icon.d.ts#L8

File: shared/common-adapters/plain-input.d.ts (hunk 1 above)
Comment (hyperobject) [reply shared/common-adapters/plain-input.d.ts]: Is there something I need to fix to make sure mobile supports this?

Comment (hyperobject) [reply shared/common-adapters/plain-input.d.ts]: I _think_ mobile should support this now. lmk if I messed something up with the .native file

Comment (hyperobject) [reply shared/common-adapters/plain-input.d.ts]: tested on ios and android, everything seems to work fine

**Pull request merged**